src/step.ts

```diff
@@ -82,54 +85,35 @@ export const emptyExecution: Execution = {
   process: [],
 }
 
-// ACTION
-interface ActionBase {
-  type: string
-  chainId: number
-  amount: string
-  token: Token
-}
-
-export interface DepositAction extends ActionBase {
-  type: 'deposit'
-}
+// STEP
+export type StepType = 'swap' | 'cross' | 'lifi'
+export type StepTool = string
 
-export interface WithdrawAction extends ActionBase {
-  type: 'withdraw'
-  toAddress: string
-  slippage: number
+export interface StepBase {
```

Comment:
Is it possible to add an optional id?

see https://github.com/lifinance/types/pull/2